src/class_loader_core.cpp

```diff
@@ -352,7 +360,7 @@ void revivePreviouslyCreateMetaobjectsFromGraveyard(
   const std::string & library_path, ClassLoader * loader)
 {
   std::lock_guard<std::recursive_mutex> b2fmm_lock(getPluginBaseToFactoryMapMapMutex());
-  MetaObjectVector & graveyard = getMetaObjectGraveyard();
+  MetaObjectGraveyardVector & graveyard = getMetaObjectGraveyard();
```

Comment:
this does not need to be protected with `getMetaObjectGraveyardMutex` because of `b2fmm_lock`?

Reply:
Ah, It should be protected. I will use the `getPluginBaseToFactoryMapMapMutex` for `MetaObjectGraveyardVector` instead of importing a new mutex.